templates/zerver/accounts_send_confirm.html

```diff
@@ -18,7 +18,7 @@ <h1>{{ _("Thanks for signing up!") }}</h1>
             </div>
 
             <div class="white-box">
-                <p>{{ _("Check your email so we can get started.") }}</p>
+                <p>{% trans %}Check your email (<b style="font-weight: 600;">{{ email }}</b>) so we can get started.{% endtrans %}</p>
```

Comment:
We should use the `z-link` approach to avoid having the precise styling content be present in the translateable string -- otherwise anytime we change the styles, translators will have to retranslate the string.

We should also use a CSS class rather than an inline style... I'm not sure why `font-weight: 600` wouldn't be the default for `<strong>`  anyway, though.

Reply:
`<strong>` is actually `700` which appears too bold. Added it as a class. I don't think we use `z-link` anywhere in the `html` files, so I left it out.

Reply:
OK; I forgot that this was a backend template. We can do something, but it's not as nice; probably this is mostly fine.